### ISSUE TITLE
Is winnappdriver 1.2.99.0 compatible with appium 1.22

### ISSUE THREAD

**Issue by OleksiiKovalnohov** [microsoft/WinAppDriver#1841]
I’m using WinnAppDriver in scope Appium (Appium version 1.22,3 winappdriver automatically installed with appium package 1.2.0)
Currently I’m investigation update winappdriver by self install without appium update
when I download and install  winappdriver v1.2.1 in same directory all is ok
but when I download and install latest experimental 1.2.99.0 I always see error
`[HTTP] --> POST /wd/hub/session
[HTTP] {“desiredCapabilities”:{“app”:“C:\myapp.exe”,“appWorkingDir”:“C:\myapp\”,“launchDelay”:“25”,“newCommandTimeout”:5000,“appArguments”:"–8BE01671F72B0219F929E4B0192089FA",“platformName”:“Windows”,“deviceName”:“Windows Device”}}
[Appium] Appium v1.22.3 creating new WindowsDriver (v1.19.1) session
[Appium] Applying relaxed security to ‘WindowsDriver’ as per server command line argument. All insecure features will be enabled unless explicitly disabled by --deny-insecure
[BaseDriver] The following capabilities were provided, but are not recognized by Appium:
[BaseDriver] launchDelay
[BaseDriver] Session created with session id: c006f9dd-03e7-4e3c-9344-bc706707573d
[WinAppDriver] WinAppDriver exists, but the checksum did not match. Was it replaced manually?
[WinAppDriver] Spawning ‘C:\Program Files (x86)\Windows Application Driver\WinAppDriver.exe’ with args: [“4724/wd/hub”]
[WD Proxy] Got response with status 500: {“status”:13,“value”:{“error”:“unknown error”,“message”:“An unknown error occurred in the remote end while processing the command.”}}
[WD Proxy] Got response with status 500: {“status”:13,“value”:{“error”:“unknown error”,“message”:“An unknown error occurred in the remote end while processing the command.”}}
[WD Proxy] Got response with status 500: {“status”:13,“value”:{“error”:“unknown error”,“message”:“An unknown error occurred in the remote end while processing the command.”}}
[WD Proxy] Got response with status 500: {“status”:13,“value”:{“error”:“unknown error”,“message”:“An unknown error occurred in the remote end while processing the command.”}}
[WD Proxy] Got response with status 500: {“status”:13,“value”:{“error”:“unknown error”,“message”:“An unknown error occurred in the remote end while processing the command.”}}
[WD Proxy] Got response with status 500: {“status”:13,“value”:{“error”:“unknown error”,“message”:“An unknown error occurred in the remote end while processing the command.”}}
[WD Proxy] Got response with status 500: {“status”:13,“value”:{“error”:“unknown error”,“message”:“An unknown error occurred in the remote end while processing the command.”}}
[WD Proxy] Got response with status 500: {“status”:13,“value”:{“error”:“unknown error”,“message”:“An unknown error occurred in the remote end while processing the command.”}}
[WD Proxy] Got response with status 500: {“status”:13,“value”:{“error”:“unknown error”,“message”:“An unknown error occurred in the remote end while processing the command.”}}
[WD Proxy] Got response with status 500: {“status”:13,“value”:{“error”:“unknown error”,“message”:“An unknown error occurred in the remote end while processing the command.”}}
[WD Proxy] Got response with status 500: {“status”:13,“value”:{“error”:“unknown error”,“message”:“An unknown error occurred in the remote end while processing the command.”}}
[WinAppDriver] WinAppDriver exited with code null, signal SIGTERM
[HTTP] <-- POST /wd/hub/session 500 10145 ms - 236
[HTTP]`

**Comment by pm90pl** [comment]
Yeah... It's a good question. I guess, no cause WAD is not compliance with W3C WebDriver standard yet and Appium Windows Driver might send request in W3C standard.
